CovertInfiltration/Localization/CovertInfiltration.int

```diff
@@ -480,10 +400,10 @@ strActivityChainsHeader=Activity Chains
 strActivityChainsBody="<Bullet/> An Activity Chain is a set of activities that must be completed in sequence to gain a large reward at the end of the chain. These activities can include Assault Missions, Covert Infiltrations, and Covert Actions.<br/><Bullet/> While the main reward(s) are usually given at the end of the chain, the earlier missions are still beneficial for XP and loot and often include secondary rewards. <br/><Bullet/> Activity Chain stages are semi-randomized, the structure of each chain and the final stage is usually the same but the mission types and rewards of the prep stages will be varied.<br/><Bullet/> Activity Chains are discovered over time by Resistance Informants and presented to XCOM for analysis. For more info, click the View Chain button at the top of the screen.<br/><br/>BRADFORD'S RECOMMENDATION: <i>Commander, we don't have the luxury of hitting ADVENT's big targets right away. We'll need to gather intel and distract the enemy before moving in on our main objective.</i>"
 
 strResistanceInformantsHeader=Resistance Informants
-strResistanceInformantsBody="<Bullet/> In the top right corner of this screen, you can see the current number of Informants. These resistance sympathizers, hidden in ADVENT controlled areas, are the key to XCOM's global operations.<br/><Bullet/> Informants gather data and report back to XCOM, revealing new opportunities for our soldiers to investigate. With our current number, you can expect our informants to uncover a new activity chain about every two weeks.<br/><Bullet/> The ADVENT administration in this region is already completely penetrated. If you want to increase the number of resistance Informants at your disposal, you will need to contact new regions or build more radio relays.<br/><br/>BRADFORD'S RECOMMENDATION: <i>Commander, we should make extending the Resistance Network a high priority. Without ADVENT targets to hit, we will run low on essential supplies and deprive the troops of fighting experience.</i>"
+strResistanceInformantsBody="<Bullet/> In the top right corner of this screen, you can see the current number of informants. These resistance sympathizers, hidden in ADVENT controlled areas, are the key to XCOM's global operations.<br/><Bullet/> Informants gather data and report back to XCOM, revealing new opportunities for our soldiers to investigate. With our current number, you can expect our informants to uncover a new activity chain about every two weeks.<br/><Bullet/> The ADVENT administration in this region is already completely penetrated. If you want to increase the number of resistance informants at your disposal, you will need to contact new regions or build more radio relays.<br/><br/>BRADFORD'S RECOMMENDATION: <i>Commander, we should make extending the Resistance Network a high priority. Without ADVENT targets to hit, we will run low on essential supplies and deprive the troops of fighting experience.</i>"

```

Comment:
> The ADVENT administration in this region is already completely penetrated

I think it should be "in the **current** region" since the player may not open the chains overview screen until after first contact